packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -175,6 +175,37 @@ class ExpressionEditor extends React.Component<Props> {
             <div className="perseus-widget-expression-editor">
                 <h3 className="expression-editor-h3">Global Options</h3>
 
+                <div className="perseus-widget-row">
+                    <label>
+                        Visible label:{" "}
+                        <TextInput
+                            value={this.props.visibleLabel}
+                            onChange={this.change("visibleLabel")}
+                        />
+                    </label>
+                    <InfoTip>
+                        <p>
+                            Optional visible text; strongly encouraged to help
+                            learners using dictation software, but can be
+                            omitted if the surrounding content provides enough
+                            context.
+                        </p>
+                    </InfoTip>
+                </div>
+
+                <div className="perseus-widget-row">
+                    <label>
+                        Aria label:{" "}
+                        <TextInput
+                            value={this.props.ariaLabel}
+                            onChange={this.change("ariaLabel")}
+                        />
+                    </label>
+                    <InfoTip>
+                        <p>Label text that's read by screenreaders.</p>
```

Comment:
I'd like to avoid that until we know it's stable and socialize expectations around aria-labels to content creators.

Reply:
Ticket to follow up: https://khanacademy.atlassian.net/browse/LEMS-2212

Reply:
++ I agree with that. Can we in the interum update the InfoTip to say:

> Label text that's read by screen readers. Highly recommend adding a label here to ensure your exercise is accessible. For more information on writting accessible labels, please see [this article](https://www.w3.org/WAI/tips/designing/#ensure-that-form-elements-include-clearly-associated-labels) or reach out to the [accessibility guild](https://khanacademy.slack.com/archives/C0A4SN2NQ)

Reply:
I updated the text and added the WCAG link. I didn't add the Slack link since this is a public project and only internal KA folks can access that channel.